news/2018-08-17-the-qat-gazette-july-2018.md

```diff
@@ -0,0 +1,97 @@
+
++ **The audio file of a song should not be artificially extended in order to meet a time limitation in the mapset section of this criteria.** This can include (but is not limited to) looping sections of the audio file, lowering the bpm of the song or section of the song, or adding small amounts of music to the song without incorporating it throughout the entire song. This does not apply to song compilations or audio files less than the minimum rankable mapset length.
+
++ **Song compilations should incorporate 3 or more songs.** Using only 2 songs in a compilation is a lackluster experience for players, and should be broken up into separate mapsets. Exceptions can be made for songs that were exclusively released together.
+
++ **Song compilations should be mixed properly and should not include abrupt breaks or long fades between different songs.** The songs used for the compilation should be similar in audio quality, volume and length. This is to ensure compilations achieve the same cohesive gameplay experience as other beatmaps.
+
+### Changes to Beatmap Veto Mediation
+
+The process of veto mediation is frequently a painful one for all involved, highlighting a lack of agreement between a Beatmap Nominator and a mapper. For the Quality Assurance Team, the mediation process is also rather time-consuming and it's disheartening to know that one of the parties involved is never going to be satisfied with the result.
+
+Whilst the existence of the beatmap veto is currently a “necessary evil”, the current turnaround time of 72 hours has proven to be quite a headache for us as there was often not enough time for an appropriate deliberation. We have therefore decided to trial changing this to allow **one week for mediation**, with the rest of the process remaining unchanged for now.
+
+### Quality Assurance Team Public Meeting #3
+
+As in previous months, the Quality Assurance Team will be hosting its third public meeting in the [osu!dev Discord server](https://discord.gg/ppy) this **Sunday the 19th of August at 1400 UTC+0**. Anyone is able to join in order to follow and participate in discussion, so if you want your input on the future directions of the mapping and modding community heard, don’t miss out!
+
+During this meeting, we will discuss the current state of hybrid (multiple game mode) mapsets and possible changes to their implementation, as well as the future of the Metadata Helpers group after disbanding the Quality Assurance Team's metadata branch.
+
+## Summary
+
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Hopefully you've found something to interest you in this month's mapping and modding news, but we must now turn our attention swiftly back to the present. There will always be more exciting times to come and we look forward to what the rest of the year may hold — even if for some it means the return to school and university. There's more to life than circles and sliders after all!
```

Comment:
Hopefully -> Hopefully,

Reply:
When I speak this a pause doesn't flow very well, so I prefer to omit the punctuation